packages/bierzo-wallet/src/logic/transactions/types/BwSendTransaction/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Address, BlockchainConnection, ConfirmedTransaction, SendTransaction } from '@iov/bcp';
+import * as React from 'react';
+
+import { BwTransaction } from '../../BwTransaction';
+import SendTransactionComponent, { BwSendTransactionProps } from './ui';
+
+export class BwSendTransaction extends BwTransaction<SendTransaction, BwSendTransactionProps> {
+  public async parse(
+    conn: BlockchainConnection,
+    trans: ConfirmedTransaction<SendTransaction>,
+    currentUserAddress: Address,
+  ): Promise<BwSendTransactionProps> {
+    const payload = trans.transaction;
+
+    const header = await conn.getBlockHeader(trans.height);
+    const time = header.time;
+
+    const received = payload.recipient === currentUserAddress;
+
+    return {
+      kind: trans.transaction.kind,
+      id: trans.transactionId,
+      memo: payload.memo,
+      amount: payload.amount,
+      time,
+      received,
+      recipient: payload.recipient,
+      sender: payload.sender,
+      success: true,
+    };
+  }
+
+  public graphicalRepresentation(sendTx: BwSendTransactionProps): JSX.Element {
+    return <SendTransactionComponent key={sendTx.id} sendTx={sendTx} />;
+  }
+
+  public csvRepresentation(tx: BwSendTransactionProps): string {
+    const parties = `"${tx.id}";"${tx.recipient}";"${tx.sender}";`;
+    const payment = `"${tx.amount.quantity}";"${tx.amount.fractionalDigits}";"${tx.amount.tokenTicker}";`;
+    const date = `"${tx.time.toISOString()}";`;
+    const status = `"${tx.received}";"${tx.success}";"${tx.err}";"${tx.memo}"`;
+
+    const txRow = `${parties}${payment}${date}${status}`;
+
+    return txRow;
```

Comment:
Can you create an array of cells then then use `cells.join(';')` for readability?

Reply:
It is already done:
```
function onDownloadCSV(): void {
    // TODO UPDATE HEADER WHEN OTHER TX TYPES ARE ADDED
    const csvHeader =
      '"ID";"Recipient";"Sender";"Quantity";"Fractional Digits";"Token Ticker";"Time";"Received";"Success";"Error";"Note"';
    const csvBody = orderedTxs.map((tx: ParsedTx<{}>) => BwTransactionFactory.getCsvRepresentation(tx));
    ....
}
```

Or... what do you mean?

Reply:
I mean the function where I commented:

Like this:

```ts
  public csvRepresentation(tx: BwSendTransactionProps): string {
    const parties = [`"${tx.id}"`, `"${tx.recipient}"`, `"${tx.sender}"`];
    const payment = [`"${tx.amount.quantity}"`, `"${tx.amount.fractionalDigits}"`, `"${tx.amount.tokenTicker}"`];
    const date = [`"${tx.time.toISOString()}"`];
    const status = [`"${tx.received}"`, `"${tx.success}"`, `"${tx.err}"`, `"${tx.memo}"`];

    const txRow = [...parties, ...payment, ...date, ...status];

    return txRow.join(';');
  }
```